src/sort/mod.rs

```diff
@@ -56,6 +58,13 @@ pub trait Sort: Any + Send + Sync + Debug {
         false
     }
 
+    /// Return the inner values and sorts.
+    /// Only eq_container_sort need to implement this method,
+    fn inner_values(&self, value: &Value) -> Vec<(&ArcSort, Value)> {
+        let _ = value;
+        vec![]
+    }
+
```

Comment:
I added this so that we can properly handle e-classes inside of collections. If we just use `extract` (what I tried previously) then it will only give us a particular e-node from a collection, instead of referencing the full e-class.